CMakeLists.txt

```diff
@@ -127,10 +127,15 @@ set(sleigh_ADDITIONAL_PATCHES "${CMAKE_CURRENT_SOURCE_DIR}/patches/sleigh/x86-ia
 # GHIDRA SLEIGH
 FetchContent_Declare(sleigh
   GIT_REPOSITORY https://github.com/lifting-bits/sleigh.git
-  GIT_TAG 5ee2f2c16250a6529108e6a6fff89e0f147502d2
+  GIT_TAG 3e188779e13935dd14790acfc2faf850810ca943
```

Comment:
This hasn't been merged into `master` yet. Do not merge until https://github.com/lifting-bits/sleigh/pull/105 has gone in.

Reply:
It's merged in now so we can proceed with this.